lms/djangoapps/discussion_api/api.py

```diff
@@ -556,6 +552,12 @@ def _handle_voted_field(form_value, cc_content, api_content, request, context):
     )
 
 
+def _handle_read_field(form_value, user, cc_content):
+    """mark thread as read for the user"""
+    if form_value:
+        user.read(cc_content)
```

Comment:
Agreed.  I think for now it makes sense to stick with the implicit unread functionality.